host_vars/pruner/public.yml

```diff
@@ -0,0 +1,44 @@
+
+FORWARD_MAIL_RELAY_MAIL_TO: "ops@example.com"
+FORWARD_MAIL_SMTP_FROM: "{{ FORWARD_MAIL_RELAY_MAIL_TO }}"
+
+TARSNAPPER_CONF: /etc/tarsnapper.yml
+
+# Can be changed to normal tarsnapper once we merge this branch
+# and code winds up on PIP
+TARSNAPPER_VERSION: git+https://github.com/open-craft/tarsnapper@jbzdak/tarsnapper-extensions
```

Comment:
I don't see any upstream PR for this change (and also no tests?):
* https://github.com/open-craft/tarsnapper/commit/7f2a7ee17fe5451af580d7aeab7719cb58d8a19f
* https://github.com/miracle2k/tarsnapper/pulls